lambdanotes/src/components/SIngleNoteView.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getNote, deleteNote } from '../actions';
+import Edit from './Edit';
+import { Link } from 'react-router-dom';
+
+
+class SingleNoteView extends Component {
```

Comment:
Very good. Seems to be a typo in the name of this file however, `SIngleNoteView.js`, or was that meant to be like that? It would be very easy to miss that when importing it.